content/ops/contingency-plan.md

```diff
@@ -0,0 +1,122 @@
+  * If the disruption is significantly affecting the delivery of our services to customers, team leads will coordinate with the 18F commsquad on how we should communicate this externally, including potentially using @18F Twitter and later following up with an 18F blog post.
+* Our core services and customer services are online and functional (we maintain availability). The team should be able to fix this (and the following items on this list) if all of the above are functioning.
+  *  cloud.gov supports customers doing self-service modifications and updates to their systems, such as the web user interface and command line login.
+  *  All of our systems are online and functional (full cloud.gov website, etc.).
+
+---
+
+## Contingency roles, responsibilities, and contact information
+
+Reference: [team roster sheet](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit) with names and ordinary team roles.
+
+Roles we probably need to define for this plan (based on [https://github.com/18F/compliance-docs/blob/master/CP-Policy.md#4-roles-and-responsibilities](https://github.com/18F/compliance-docs/blob/master/CP-Policy.md)):
+
+* Development Operations
+* Disaster Recovery Team
+* Senior Director
+* Director of Compliance
+
+People who need to get notified when there's a serious problem:
+
+If this is not a security incident and just unplanned downtime:
+
+* cloud.gov team
+* Infrastructure and @commsquad
+* Key 18F staff (comms lead, director of infrastructure, etc.)
+* cloud.gov users (through statuspage)
+* Designated FedRAMP personnel (when applicable)
+
+If this is a security incident first follow the IR guide, then notify all of the above plus:
+* GSA secops team
+
+---
+
+## Risk management practices that support restoring key functions
+
+If something goes wrong, we have these continuously maintained resources to support our work:
+
+* GitHub backup
+* We have access to all of our data in GitHub even if GitHub is offline/unavailable/compromised.
+* Team contact info backup
+* We maintain a [team roster](https://docs.google.com/spreadsheets/d/1mW3tphZ98ExmMxLHPogSpTq8DzYr5Oh8_SHnOTvjRWM/edit).
+* Each team member core to cloud.gov operations maintains a list of the phone numbers of the other core team members, stored offline in a protected way (such as in the contacts app in their GSA phone or in a file on their GSA laptop).
+* Thorough documentation
+* The basic steps for rebuilding cloud.gov are written in our [cg-provision repository](https://github.com/18f/cg-provision).
+* We back take snapshots of all RDS instances and store them for at least 30 days.
+* For AWS availability we use hot multi-AZ replication. By doing that we inherit AWS controls for alternate processing.
```

Comment:
I'd specify what "AZ" means, and link to the inherited AWS controls if possible.